pkg/cloud/azure/actuators/clients.go

```diff
@@ -36,7 +36,7 @@ func CreateOrUpdateNetworkAPIServerIP(scope *Scope) {
 	if scope.Network().APIServerIP.Name == "" {
 		h := fnv.New32a()
 		h.Write([]byte(fmt.Sprintf("%s/%s/%s", scope.SubscriptionID, scope.ClusterConfig.ResourceGroup, scope.Cluster.Name)))
-		scope.Network().APIServerIP.Name = strings.ToLower(azure.DefaultPublicIPPrefix + fmt.Sprintf("%x", h.Sum32()))
+		scope.Network().APIServerIP.Name = azure.GeneratePublicIPName(scope.Cluster.Name)
```

Comment:
Can we use the hash as an input as well? the reason for attaching the hash is to generate a unique name which can be used as a prefix for dnsname, otherwise if multiple people create clusters with same name, will have conflict with dnsname

Reply:
Ahhh, that makes sense. Fixed!